client/views/users/detail.html

```diff
@@ -82,6 +82,15 @@ <h1 ng-if="user.data.id" class="page-header">
              label="{{'USER_DETAIL_NOTES' | translate}}"
              model="user.data.notes"></field>
 
+      <field name="organization" type="organization" required="true"
+             label="{{'USER_DETAIL_ORGANIZATION' | translate}}"
+             model="user.data.organization"
+             on-select="user.organizationChanged()"></field>
+
+      <div ng-if="user.showOrganizationWarning" class="alert alert-danger" translate>
```

Comment:
If we directly change the role, it makes sense to show the warning before changing in case the role is not user